private/buf/cmd/buf/command/push/push.go

```diff
@@ -278,23 +278,21 @@ func validateLabelFlags(flags *flags) error {
 // We do not allow overlaps between `--label`, `--tag`, `--branch`, and `--draft` flags
 // when calling push. Only one type of flag is allowed to be used at a time when pushing.
 func validateLabelFlagCombinations(flags *flags) error {
-	if len(flags.Labels) > 0 && len(flags.Tags) > 0 {
-		return appcmd.NewInvalidArgumentErrorf("--%s and --%s (-%s) cannot be used together.", labelFlagName, tagFlagName, tagFlagShortName)
-	}
-	if len(flags.Labels) > 0 && flags.Branch != "" {
-		return appcmd.NewInvalidArgumentErrorf("--%s and --%s cannot be used together.", labelFlagName, branchFlagName)
+	var usedFlags []string
+	if len(flags.Labels) > 0 {
+		usedFlags = append(usedFlags, labelFlagName)
 	}
-	if len(flags.Labels) > 0 && flags.Draft != "" {
-		return appcmd.NewInvalidArgumentErrorf("--%s and --%s cannot be used together.", labelFlagName, draftFlagName)
+	if len(flags.Tags) > 0 {
+		usedFlags = append(usedFlags, tagFlagName)
 	}
-	if len(flags.Tags) > 0 && flags.Branch != "" {
-		return appcmd.NewInvalidArgumentErrorf("--%s (-%s) and --%s cannot be used together.", tagFlagName, tagFlagShortName, branchFlagName)
+	if flags.Branch != "" {
+		usedFlags = append(usedFlags, branchFlagName)
 	}
-	if len(flags.Tags) > 0 && flags.Draft != "" {
-		return appcmd.NewInvalidArgumentErrorf("--%s (-%s) and --%s cannot be used together.", tagFlagName, tagFlagShortName, draftFlagName)
+	if flags.Draft != "" {
+		usedFlags = append(usedFlags, draftFlagName)
 	}
-	if flags.Draft != "" && flags.Branch != "" {
-		return appcmd.NewInvalidArgumentErrorf("--%s and --%s cannot be used together.", draftFlagName, branchFlagName)
+	if len(usedFlags) > 1 {
+		return appcmd.NewInvalidArgumentErrorf("These flags cannot be used in combination with one another: %s", usedFlags)
```

Comment:
```
Failure: These flags cannot be used in combination with one another: [label tag branch draft]
```

We might want to craft this error message so it is a string joined by `, ` with `--` before each flag.

Reply:
That's a good call, put in the fix :)